examples/hello_world.py

```diff
@@ -7,3 +7,26 @@
 
 submitter = ArgoSubmitter()
 result = couler.run(submitter=submitter)
+
+# Instead of passing the submitter each time, you can just set it as default!
+ArgoSubmitter.set_default(submitter)
```

Comment:
This syntax looks weird to me. Why do we need ArgoSubmitter and then use submitter again?

Reply:
I've created a different example file that's easier to understand.